spec/importc.dd

```diff
@@ -32,8 +32,9 @@ $(H2 $(LNAME2 dialect, ImportC Dialect))
     )
 
     $(P Adjustment to the ImportC dialect is made to match the
-    behavior of the C compiler that the D compiler is matched to. For
-    example, on Win32 D is matched to the Digital Mars C compiler,
+    behavior of the C compiler that the D compiler is matched to
+    (aka the $(I associated C compiler))
```

Comment:
```suggestion
    i.e. the $(I associated C compiler).
```
i.e. is better than aka here, and the sentence should end after the parenthesis, currently it runs straight onto the example.